docs/general/builtins/scalar.rst

```diff
@@ -508,6 +508,49 @@ Example::
    In both cases, the scalar functions lpad and rpad, do now accept a len
    greater than 50000.
 
+.. _scalar-encode:
+
+``encode(bytea, format)``
+-------------------------
+
+Encode takes a binary string in hex format and returns a textual representation
+into the specified format. Supported formats are base64, hex, and escape. The
+escape format represents unprintable characters with an octal sequence `\nnn`.
+
+Synopsis::
+
+    encode(string1, format)
+
+Example::
+
+   cr> select encode(E'123\b\t56', 'base64');
+   +----------------------------------------+
+   | select encode(E'123\b\t56', 'base64'); |
+   +----------------------------------------+
+   | MTIzCAk1Ng==                           |
+   +----------------------------------------+
```

Comment:
All examples which uses the `cr>` prefix are executed and tests when running our `itest`  gradle task.
I think this will fail as the output `SELECT 1 row in set (... sec)` is missing.
Another minor: all lines should be intended by 4 spaces.